src/d2r-map.ahk

```diff
@@ -1,22 +1,28 @@
-#SingleInstance, Force
+;#SingleInstance, Force
 #Persistent
+if not A_IsAdmin {
```

Comment:
I've tried to avoid forcing to run as admin and leaving it to the user to run as admin if required. Is there a reason this is needed?

Reply:
As far as I was aware the app HAS to run in admin so that it can read from a program that is in escalated memory.
Since I can't use the script without the escalation, I might as well add it. QOL, once less step. it doesnt't need to be there though  :p